pkg/cloud/aws/services/certificates/service.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package certificates
+
+import (
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud/aws/actuators"
+)
+
+// Service holds a collection of interfaces.
```

Comment:
I see this comment copy and pasted around and I don't think it's actually accurate anymore. Please replace this whole section of comment describing what the certificate service provides

Reply:
Updated to
```
// Service groups certificate related operations together and allows
// certificate updates to be applied to the actuator scope.
```